### PR TITLE
[Tool] Log access flask server

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
This patch aims to provide a way to quickly
access log dirs from remote server using a flask
server.

Fixes: #906

Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: tools/log_server/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Flask Log Server
+
+Logs are important for two purposes, when we are developing the code and when the code fails and we want to find what went wrong. To access the logs quickly, the flask server tool was created. With a couple of changes in the dev env, users can run a poor man's flask server which server the redant log dir.
```
Comment (nik-redhat) [inline tools/log_server/README.md:3]: nit: in the last line -> "which serves the redant log..."

File: tools/log_server/log_access.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from flask_autoindex import AutoIndex
+import configparser
+
+app = Flask(__name__)
+
+if __name__ == "__main__":
+    config = configparser.ConfigParser()
+    config.read('log_access.ini')
```
Comment (nik-redhat) [inline tools/log_server/log_access.py:9]: Also, is there a fail check being done in the library? If not, I think we should be doing here explicitly to fail safely.

Comment (srijan-sivakumar) [reply tools/log_server/log_access.py:9]: Ah..ack.

**Comment by srijan-sivakumar** [comment]
```console
redant-demo) [root@rhsdev-virt05 log_server]# python3 log_access.py 
 * Serving Flask app 'log_access' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://10.70.40.5:5000/ (Press CTRL+C to quit)
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**